### PR TITLE
Ceph - first version

### PR DESCRIPTION
**Pull request by fernandosanchezmunoz**
Ceph based on the ceph-on-mesos framework https://github.com/vivint-smarthome/ceph-on-mesos by @timcharper

### CODE REVIEW

**Comment by fernandosanchezmunoz** [comment]
associated tutorial: https://github.com/dcos/examples/pull/25

File: repo/packages/C/ceph/0/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "packagingVersion": "3.0",
+  "name": "ceph",
+  "version": "0.2.9-0.1",
+  "scm": "https://github.com/ceph",
+  "maintainer": "support@mesosphere.io",
+  "website": "https://www.ceph.org",
+  "description": "Ceph, a free-software storage platform, implements object storage on a single distributed computer cluster, and provides interfaces for object-, block- and file-level storage. Ceph aims primarily for completely distributed operation without a single point of failure, scalable to the exabyte level, and freely available.\n\nCeph replicates data and makes it fault-tolerant,[4] using commodity hardware and requiring no specific hardware support. As a result of its design, the system is both self-healing and self-managing, aiming to minimize administration time and other costs.",
```
Comment (ryadav88) [inline repo/packages/C/ceph/0/package.json]: [4] -> for

File: repo/packages/C/ceph/0/package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "tags": [
+    "ceph",
+    "storage",
+    "filesystem"
+    ],
+  "preInstallNotes": "This DC/OS Service is currently EXPERIMENTAL. There may be bugs, incomplete features, incorrect documentation, or other discrepancies.\n\nPLEASE MAKE SURE to check the online tutorial at https://github.com/fernandosanchezmunoz/dcos-ceph/ \n\n Also PLEASE MAKE SURE to set correctly the `cluster_network` and `public_network` fields in the Advanced Installation menu.",
```
Comment (ryadav88) [inline repo/packages/C/ceph/0/package.json]: Add a link to the example at the end of `preInstallNotes`

File: repo/packages/C/ceph/0/package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  "postInstallNotes": "Service installed.\n\nPlease access the UI through the endpoint created in Marathon-LB.",
+  "postInstallNotes": "Service installed.\n\nPlease access the UI through the endpoint created in Marathon-LB.",
```
Comment (ryadav88) [inline repo/packages/C/ceph/0/package.json]: duplicate

File: repo/packages/C/ceph/0/config.json [hunk 4]
```diff
@@ -0,0 +1,82 @@
+{
```
Comment (ryadav88) [inline repo/packages/C/ceph/0/config.json]: I think we should also explain the steps in prereqs in dcos/example along with screenshots.

File: repo/packages/C/ceph/0/package.json (hunk 1 above)
Comment (fernandosanchezmunoz) [inline repo/packages/C/ceph/0/package.json]: corrected.

File: repo/packages/C/ceph/0/package.json (hunk 2 above)
Comment (fernandosanchezmunoz) [inline repo/packages/C/ceph/0/package.json]: done, linked to what should be the tutorial URL: https://github.com/dcos/examples/tree/master/1.8/ceph

File: repo/packages/C/ceph/0/package.json (hunk 3 above)
Comment (fernandosanchezmunoz) [inline repo/packages/C/ceph/0/package.json]: whoops. removed.

File: repo/packages/C/ceph/0/config.json (hunk 4 above)
Comment (fernandosanchezmunoz) [inline repo/packages/C/ceph/0/config.json]: well everything seems pretty explained in detail with copy&paste-ready code to me. What's missing here? https://github.com/fernandosanchezmunoz/examples/blob/c1446aced1ff8096693c9cc6a5e1b7e09d413d2a/1.8/ceph/README.md

Comment (ryadav88) [inline repo/packages/C/ceph/0/config.json]: 👍  read the full example now.

**Pull request merged**